### PR TITLE
Expose Route property on Request object

### LINKED ISSUE #6184: Expose Route property on HttpRequest through Liquid
Similar to the way the `HttpRequest` object is exposed via properties like `QueryString`, `Query`, `Cookies` etc, we should expose `Route`. Which becomes extra useful in combination with #5305

### PR DESCRIPTION
**Pull request by sfmskywalker**
Fixes #6184

### CODE REVIEW

**Comment by agriffard** [comment]
Can you please add some documentation?

**Comment by sfmskywalker** [comment]
I always forget about documentation 😅  Will add 👍

File: src/OrchardCore/OrchardCore.DisplayManagement.Liquid/RequestLiquidTemplateEventHandler.cs [hunk 1]
```diff
@@ -117,5 +120,15 @@ public HttpContextItemsWrapper(IDictionary<object,object> items)
                 Items = items;
             }
         }
+
+        private class RouteValueDictionaryWrapper
+        {
+            public readonly IReadOnlyDictionary<string, object> RouteValueDictionary;
```
Comment (hishamco) [inline src/OrchardCore/OrchardCore.DisplayManagement.Liquid/RequestLiquidTemplateEventHandler.cs:126]: Why it's setter?

**Comment by agriffard** [comment]
@sfmskywalker Ok, just waiting for some docs to merge it ;-)

**Comment by agriffard** [comment]
Please add some docs @sfmskywalker so we can merge.

**Comment by jtkech** [comment]
Just to say that the doc already exists for the httpcontext request properties

We only need to add the line related to this new property

**Comment by sfmskywalker** [comment]
Damnit, thanks for reminding me Antoine 😅

File: src/OrchardCore/OrchardCore.DisplayManagement.Liquid/RequestLiquidTemplateEventHandler.cs (hunk 1 above)
Comment (sfmskywalker) [reply src/OrchardCore/OrchardCore.DisplayManagement.Liquid/RequestLiquidTemplateEventHandler.cs:126]: @hishamco I don't understand your question. Can you rephrase please?

**Comment by sfmskywalker** [comment]
Docs updated ✔️

**Comment by agriffard** [comment]
Thank you @sfmskywalker
